api/build.gradle

```diff
@@ -371,105 +370,105 @@ dependencies {
     __swaggerCodegenV2__ "io.swagger:swagger-codegen-cli:$project.ext.SWAGGER_2_CODEGEN_VERSION"
     __swaggerCodegenV3__ "io.swagger.codegen.v3:swagger-codegen-cli:$project.ext.SWAGGER_3_CODEGEN_VERSION"
 
-    compile 'ch.qos.logback:logback-classic:1.2.11'
-    compile "com.fasterxml.jackson.core:jackson-annotations:$project.ext.JACKSON_VERSION"
-    compile "com.fasterxml.jackson.core:jackson-core:$project.ext.JACKSON_VERSION"
-    compile "com.fasterxml.jackson.core:jackson-databind:$project.ext.JACKSON_DATABIND_VERSION"
-    compile "com.fasterxml.jackson.dataformat:jackson-dataformat-yaml:$project.ext.JACKSON_VERSION"
-    compile "com.fasterxml.jackson.module:jackson-module-kotlin:$project.ext.JACKSON_VERSION"
-    compile "com.github.java-json-tools:json-patch:1.13"
-    compile "com.github.rholder:guava-retrying:2.0.0"
-    compile 'org.javers:javers-core:5.15.0'
-    compile "com.squareup.okhttp:logging-interceptor:$project.ext.OKHTTP_VERSION"
-    compile "com.squareup.okhttp:okhttp:$project.ext.OKHTTP_VERSION"
-    compile "io.opencensus:opencensus-api:$project.ext.OPENCENSUS_VERSION"
-    compile "io.opencensus:opencensus-exporter-stats-stackdriver:$project.ext.OPENCENSUS_VERSION"
-    compile "io.opencensus:opencensus-exporter-trace-stackdriver:$project.ext.OPENCENSUS_VERSION"
-    compile "io.opencensus:opencensus-impl:$project.ext.OPENCENSUS_VERSION"
-    compile 'io.swagger:swagger-annotations:1.6.7'
-    compile "javax.inject:javax.inject:1"
-    compile "org.liquibase:liquibase-core:$project.ext.LIQUIBASE_VERSION"
-    compile 'mysql:mysql-connector-java:8.0.30'
-    compile "org.apache.commons:commons-collections4:4.4"
-    compile 'org.apache.commons:commons-lang3:3.12.0'
-    compile 'org.apache.commons:commons-text:1.10.0'
-    compile "org.apache.httpcomponents:httpclient:4.5.13"
-    compile "org.springframework.boot:spring-boot-starter-validation:$project.ext.SPRING_BOOT_VERSION"
-    compile "commons-codec:commons-codec:1.15"
-    compile 'com.auth0:java-jwt:3.19.2'
-    compile 'io.netty:netty-common:4.1.82.Final'
-    compile 'org.json:json:20220924'
+    api 'ch.qos.logback:logback-classic:1.2.11'
```

Comment:
"api" allows usage, whereas "implementation" is for including without direct usage, according to my understanding. Since "api" is more permissive, it's a safer default choice.

Reply:
could you say a little more about what "usage" and "include" mean in this context?